config/proposer/loader/loader.go

```diff
@@ -69,6 +69,9 @@ func WithGasLimit() SettingsLoaderOption {
 				log.Warnf("Gas limit was intentionally set to 0, this will be replaced with the default gas limit of %d", params.BeaconConfig().DefaultBuilderGasLimit)
 			}
 			rgl := reviewGasLimit(validator.Uint64(gl))
+			if &rgl != nil {
+				log.Infof("Using a global gas limit of %d if builder is enabled", rgl)
```

Comment:
maybe need to lowercase , how does this sound?

Reply:
What do you mean "if builder is enabled"? Can we check if it is enabled?